### ISSUE TITLE
This is a question, sorry, i did'nt know where to ask

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#113]
```
I have android 2.1, and i installed ebookdroid-1.3.0-dev-r702.apk.
I have been reading, sometime now, that there is an option to enable HW 
acceleration in ebookdroid, but i cant find this option in the settings.

Is it available in android 2.1?
How do i enable it?

thanks
```

Original issue reported on code.google.com by `lanos...@gmail.com` on 30 Dec 2011 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
For HW acceleration is available for Android 3.0+ only.

```

Original comment by `Andrei.K...@gmail.com` on 30 Dec 2011 at 7:36
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mc.creat` on 30 Dec 2011 at 10:03
- Changed state: **Invalid**
